### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by AliAlAlusi**
Expanded the Usage/Configuration Options section of the Gazebo Simulation guide to include the option of changing the Gazebo render engine.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks - is their an associated PX4-Autopilot PR/issue? @beniaminopozzan are you OK with this?

Note that I run this in WSL2 and haven't had any rendering issues, and I don't set this.

**Comment by beniaminopozzan** [comment]
@hamishwillee yes there is: https://github.com/PX4/PX4-Autopilot/pull/22693
100% OK with this.

File: en/sim_gazebo_gz/index.md [hunk 1]
```diff
@@ -228,6 +228,12 @@ where `ARGS` is a list of environment variables including:
   Lets PX4 know that it should not launch an instance of Gazebo.
   Gazebo will need to be launched separately, as described in [Standalone Mode](#standalone-mode).
 
+- `PX4_GZ_SIM_RENDER_ENGINE`:
+  Sets the render engine to be used by gazebo, by default it is OGRE 2.
+
+  - Can be set to OGRE 1 (`PX4_GZ_SIM_RENDER_ENGINE=ogre`) to solve rendering issues when running PX4 on a virtual machine.
+  
+
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/index.md:236]:
```suggestion
- `PX4_GZ_SIM_RENDER_ENGINE`:
  Sets the render engine to be used by gazebo.

  The default rendering engine (OGRE 2) is not well supported on some platforms/environments.
  Specify `PX4_GZ_SIM_RENDER_ENGINE=ogre` to set the rendering engine to OGRE 1 if you have rendering issues when running PX4 on a virtual machine.

```

**Comment by hamishwillee** [comment]
Thanks very much!

**Pull request merged**